app/bundles/LeadBundle/Views/Company/form_standalone.html.php

```diff
@@ -19,7 +19,34 @@
     :
     $view['translator']->trans('mautic.company.menu.new');
 $view['slots']->set('headerTitle', $header);
-
+$buttons[] = [
+    'attr' => [
```

Comment:
This results in kind of an awkward UI with the dropdown separated from the form buttons. Can we simply append this as a button in the form? https://github.com/mautic/mautic/blob/e71e70e3dd91408e80860ecac00e2d78304b34ed/app/bundles/LeadBundle/Form/Type/CompanyType.php#L95 

See https://github.com/mautic/mautic/blob/e71e70e3dd91408e80860ecac00e2d78304b34ed/app/bundles/CampaignBundle/Form/Type/CampaignType.php#L113-L113 for an example. Can use `post_extra_buttons` instead of `pre_extra_buttons`